src/main/java/at/hannibal2/skyhanni/features/inventory/bazaar/BazaarApi.kt

```diff
@@ -202,12 +240,25 @@ object BazaarApi {
     @HandleEvent(onlyOnSkyblock = true)
     fun onChat(event: SkyHanniChatEvent) {
         val message = event.message.removeColor()
-        val item = resetCurrentSearchPattern.matchMatcher(message) {
-            group("item")
-        } ?: return
-
-        if (currentSearchedItem == item) {
-            currentSearchedItem = ""
+        transactionPattern.matchMatcher(message) {
+            val item = group("item")
+            val coins = group("coins").replace(",", "").toDoubleOrNull() ?: return
```

Comment:
```suggestion
            val coins = group("coins").formatDoubleOrNull() ?: return
```
will prob need to import it